server/app/views/components/SelectWithLabel.java

```diff
@@ -101,6 +101,12 @@ public SelectWithLabel setDisabled(boolean disabled) {
     return this;
   }
 
+  @Override
+  public SelectWithLabel setReadOnly(boolean disabled) {
```

Comment:
Covariant return types :face_exhaling: Yes, this is actually needed.

Reply:
Understood.